plugins/zfs/zpool_iostat

```diff
@@ -1,4 +1,7 @@
-#!/bin/bash
+#!/bin/sh
+
+#%# family=auto
+#%# capabilities=autoconf
```

Comment:
To add "autoconf" to capabilities, the autoconf itself should be able to say both "yes" and "no", after checking for the presence of whatever's being monitored.  For now, it just says "yes", which would be wrong on any system not using zfs.